loadtest/control/simulcontroller/utils.go

```diff
@@ -47,11 +48,21 @@ func genMessage(isReply bool) string {
 	return message
 }
 
+func splitName(name string) (string, string) {
+	var prefix string
+	if strings.HasPrefix(name, "testuser-") {
```

Comment:
Now that the "testuser-" string has got repeated 4 times, I would suggest refactoring this somewhere else. I think the `user` package would be a nice place since it's got to do with users in a load test.

```go

const (
	// SharedPrefix is a common prefix shared by all users in the load test.
	SharedPrefix = "testuser-"
)
```

And then use it here and in the main package.